### PR TITLE
update readme to point to pip install instead of source

### PR DESCRIPTION
**Pull request by sroet**
Now that the pip install commands are working, we should update our installation instructions as well

### CODE REVIEW

**Comment by sroet** [comment]
The tests fail with the same issue that #138 tries to fix, so this shouldprobably be merged after that one

File: README.md [hunk 1]
```diff
@@ -35,33 +35,28 @@ Once the environment is created, activate it:
...
-python -m pip install .
+python -m pip install pytom-match-pick
 ```
 
-## Tests
-
-To run the unittests:
-
+## Cupy warning
+Trying to import the code with
```
Comment (McHaillet) [inline README.md:53]:
```suggestion
Having issues running the software? If cupy is not correctly installed, 
```

File: README.md [hunk 2]
```diff
@@ -35,33 +35,28 @@ Once the environment is created, activate it:
...
+Trying to import the code with 
 ```commandline
-cd tests
-python -m unittest discover
+python -c "import pytom_tm"
 ```
 
-If cupy is not correctly installed, this should raise some errors:
-
+can show a cupy warning. If this is the case, this probably means cupy is not correctly installed.
```
Comment (McHaillet) [inline README.md:58]:
```suggestion
will show an import error. If this is the case, this probably means cupy is not correctly installed.

```

File: README.md [hunk 3]
```diff
@@ -35,33 +35,28 @@ Once the environment is created, activate it:
...
+If your cupy installation is broken, the following should raise some errors:
```
Comment (McHaillet) [inline README.md:59]:
```suggestion
Alternatively, cupy can sometimes be installed without issue but not detect CUDA correctly. In that case, the following should raise some errors:
```

**Review by McHaillet — CHANGES_REQUESTED** [review]
Installation part is good!

For the cupy errors, some suggested changes and I also think this part,

> To solve cupy installation issues, please check [the cupy docs](https://docs.cupy.dev/en/stable/install.html#installing-cupy). It might be solved by installing a specific build compatible with the installed cuda toolkit. 

, could be changed to first trying to install with the conda cuda-toolkit and cupy option. And if that doesnt work looking at the cupy docs.

File: README.md (hunk 2 above)
Comment (sroet) [reply README.md:58]: Currently the code only does logging.warning, not an actually import error

**Comment by sroet** [comment]
@McHaillet, one of your suggestions isn't accurate with the current code behavior. I have implemented the others

File: README.md (hunk 2 above)
Comment (McHaillet) [reply README.md:58]: Fair!

**Review by McHaillet — APPROVED** [review]
LGTM!

**Pull request merged**